doc/getting_started.md

```diff
@@ -0,0 +1,30 @@
+# Getting Started
+
+The best way to get started is to run through the [tutorial](https://github.com/AlphaWallet/TokenScript/tree/master/tutorial).
+
+## Implicit Attributes
+
+There are a few implicit attributes available:
+
+* name
+* symbol
+* contractAddress
+* ownerAddress
+* tokenId
+
+`tokenId` is especially useful for defining attributes with a function origin like this:
+
+```
+<ts:attribute-type id="locality" syntax="1.3.6.1.4.1.1466.115.121.1.15">
+  <ts:origins>
+    <ts:ethereum function="getLocality" contract="EntryToken" as="utf8">
+        <ts:data>
+          <ts:uint256 ref="tokenId"/>
+        </ts:data>
```

Comment:
probably good to highlight the relevance to erc721 assets? @hboon

Reply:
@James-Sangalli added an example and reference to ERC721